crypto/hash/types.go

```diff
@@ -8,27 +8,40 @@ type HashingAlgorithm int
 const (
 	// Supported hashing algorithms
 	UnknownHashingAlgorithm HashingAlgorithm = iota
+	// SHA-2
 	SHA2_256
 	SHA2_384
+	// SHA-3
 	SHA3_256
 	SHA3_384
+	// KMAC (Keccak based MAC algorithm)
 	KMAC128
+	// legacy Keccak
+	Keccak_256
 )
 
 // String returns the string representation of this hashing algorithm.
-func (f HashingAlgorithm) String() string {
-	return [...]string{"UNKNOWN", "SHA2_256", "SHA2_384", "SHA3_256", "SHA3_384", "KMAC128"}[f]
+func (h HashingAlgorithm) String() string {
+	return [...]string{
+		"UNKNOWN",
+		"SHA2_256",
+		"SHA2_384",
+		"SHA3_256",
+		"SHA3_384",
+		"KMAC128",
+		"Keccak_256"}[h]
 }
 
 const (
 	// minimum targeted bits of security
 	securityBits = 128
 
 	// Lengths of hash outputs in bytes
-	HashLenSha2_256 = 32
-	HashLenSha2_384 = 48
-	HashLenSha3_256 = 32
-	HashLenSha3_384 = 48
+	HashLenSHA2_256   = 32
+	HashLenSHA2_384   = 48
+	HashLenSHA3_256   = 32
+	HashLenSHA3_384   = 48
+	HashLenKeccak_256 = 32
 
```

Comment:
it seems there are some dependency on this in flow-go, is it possible that there are some dependencies out there as well?

Reply:
You're right, updated in https://github.com/onflow/flow-go/pull/1865/commits/7f6a5f63123ea11d4c9a54c2e535a734532a3ffa

Reply:
there might be some more dependencies in FCL and other client sides